Lab3/ComputerStore.WebUI/Controllers/BuyersController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using ComputerStore.BusinessLogicLayer.Managers;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ComputerStore.WebUI.Controllers
+{
+    public class BuyersController : Controller
+    {
+        private readonly BuyerManager _buyerManager;
+
+        public BuyersController(BuyerManager buyerManager)
+        {
+            _buyerManager = buyerManager;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Details(int id)
```

Comment:
We cannot do anything except viewing the buyers?

Reply:
It's a question for better logic understanding and not a task for improving)
Moreover, I can't find this page on UI)

Reply:
I want to expand this controller after adding Identity.
User enter buyer info when buy product.
You can find buyer details on a order page.